channels/us.m3u

```diff
@@ -1486,13 +1486,13 @@ https://zypelive-lh.akamaihd.net/i/default_1@710948/master.m3u8
 #EXTINF:-1 tvg-id="EsperanzaTV.us" tvg-name="Esperanza TV" tvg-country="US" tvg-language="Spanish" tvg-logo="http://iptvchile.xyz/Logos/TeVe/esperanza%20tv.png" group-title="",Esperanza TV (480p)
 http://k3.usastreams.com:1935/etvSD/etvSD/playlist.m3u8
 #EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN (720p)
-https://gma2.blab.email/fs2.m3u8
+http://164.68.124.111/fs2.m3u8
 #EXTINF:-1 tvg-id="ESPN2.us" tvg-name="ESPN 2" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN 2 (720p)
-https://gma2.blab.email/espn2.m3u8
+http://164.68.124.111/espn2.m3u8
 #EXTINF:-1 tvg-id="ESPNNews.us" tvg-name="ESPN News" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Sports",ESPN News (720p)
-https://gma2.blab.email/espnews.m3u8
+http://164.68.124.111/espnews.m3u8
```

Comment:
Doesn't work.

Reply:
File technicaly exists and downloads. Let's test if it's not 24/7

Reply:
Re-verified, still a 0 octet file being sent.

Reply:
Will remove it tomorrow if nothing changes